index.html

```diff
@@ -3098,6 +3098,119 @@ <h1>Speakers</h1>
+                                            <a target="default" href="http://linkedin.com/in/suraj-narwade-85436a38"><i class="icon social_linkedin"></i></a>
+                                            <a target="default" href="https://github.com/surajnarwade"><i class="icon fa fa-github"></i></a>
+                                        </div>
+                                    </div>
+                              </div>
+                              <span class="speaker-name">Suraj Narwade</span>
+                              <span>Red Hat</span>
+                            </div>
+                        </div>
+
+                        <div class="col-md-3 col-sm-6 speaker-column">
+                            <div class="speaker">
+                              <div class="image-holder">
+                                    <img class="background-image" alt="Aayush Arora" src="img/AayushArora.jpg">
+                                    <div class="hover-state text-center preserve3d">
+                                        <div class="social-links vertical-align">
+                                            <a target="default" href="http://aayusharora.bitballoon.com/"><i class="icon fa fa-external-link"></i></a>
+                                            <a target="default" href="https://twitter.com/angularboy"><i class="icon social_twitter"></i></a>
+                                            <a target="default" href="https://www.facebook.com/angularboy"><i class="icon social_facebook"></i></a>
+                                            <a target="default" href="https://www.linkedin.com/in/angularboy/"><i class="icon social_linkedin"></i></a>
+                                            <a target="default" href="https://github.com/aayusharora"><i class="icon fa fa-github"></i></a>
+                                        </div>
+                                    </div>
+                              </div>
+                              <span class="speaker-name">Aayush Arora</span>
+                              <span>Web Developer</span>
+                              <span>FOSSASIA</span>
+                            </div>
+                        </div>
+
+                        <div class="col-md-3 col-sm-6 speaker-column">
+                            <div class="speaker">
+                              <div class="image-holder">
+                                    <img class="background-image" alt="Pronnoy Goswami" src="img/PronnoyGoswami.jpg">
+                                    <div class="hover-state text-center preserve3d">
+                                        <div class="social-links vertical-align">
+                                            <a target="default" href="https://www.linkedin.com/in/pronnoygoswami/"><i class="icon fa fa-external-link"></i></a>
```

Comment:
Remove the external link since linkedIn and external are same.

Reply:
@simsausaurabh Removed.